### DISCUSSION TITLE
In-app notifications

### DISCUSSION THREAD

**Discussion by sshanzel** [dailydotdev/daily#353] [Feature Requests]
I think it would be nice to have a notification feed to easily manage updates inside the app.

As a writer, I want to immediately get notified when someone upvoted/commented on my article. 

As a user, I want to get notified when people upvoted/commented on my comments. It would be nice to know also if my comment got featured.

In relation to [this request](https://github.com/dailydotdev/daily/discussions/352), it would be lovely to get notified as well.

**Comment by rebelchris** [comment]
Done
